irc/weechat/Portfile

```diff
@@ -1,39 +1,19 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
+PortGroup           cmake 1.1
 PortGroup           conflicts_build 1.0
-PortGroup           github 1.0
 
+conflicts           weechat-devel
```

Comment:
this is gone now, right?

Reply:
It is, but I wasn't sure how to handle the case if `weechat-devel` is currently installed. Is there a canonical way for how one should handle it?

In my particular usecase, it was a reminder for me to `port uninstall weechat-devel` before installing this version.

Reply:
You're right. You should leave create a remnant port weechat-devel using the obsolete portgroup for a year, according to tradition, and then delete it after one year. It could be done in a separate portfile if you like -- perhaps simpler than trying to manhandle it into this portfile, although that could be done too. There are existing examples of obsolete ports currently in the tree you could copy -- search for "obsolete" -- and then deleted in a year.

Reply:
Thanks. I've created a new port in the obsolete portgroup, `weechat-devel` in the same commit.

I can also split it into two commits under this PR, if that's more appropriate.